operator-common/src/test/java/io/strimzi/operator/common/OperatorMetricsTest.java

```diff
@@ -166,13 +224,68 @@ protected Status createStatus() {
                 })));
     }
 
+    @Test
+    public void testFailingReconcileWithSelectorLabel(VertxTestContext context)  {
```

Comment:
Should this be `testFailingReconcileWithoutSelectorLabel`? I do not see any label. Same in the further tests.